### DISCUSSION TITLE
Indicator for authorized fetch enabled posts

### DISCUSSION THREAD

**Discussion by noantiq** [tuskyapp/Tusky#4326] [Ideas]
Hi, I'm not too familiar with Mastodon/ActivityPub so I don't know how straight forward this would be, but it would be nice to have some indication that a link to a post might not be easily accessible because the author's instance has authorized fetch enabled. This indicator might only be relevant next to the option to share or the option to copy the link to a post.

**Comment by connyduck** [comment]
This cannot be done on client side because the Mastodon API does not expose any information related to authorized fetch.
Also I'm not sure I understand why this would be important, a post is usually accessible through its link when the visibility is public or unlisted, which has nothing to do with authorized fetch.

**Reply by noantiq** [reply]
Oh well, I just had a link to a post earlier that gave a 403 error when opening in a browser but when I opened it in Tusky while logged in I could see it and with my very limited understanding I thought this might be the problem but must have been something else then I suppose. Also when checking it again I noticed that it was a Friendica…
Anyway, my suggestion doesn‘t seem helpful then and I‘m going to close this.
